bike-erp/src/index.tsx

```diff
@@ -1,13 +1,14 @@
-import { ThemeProvider } from "@material-ui/core";
-import React from "react";
 import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
+
```

Comment:
Remove Space here

Reply:
As agreed offline, we'll keep the space to separate the import blocks and to adhere to the coding conventions 👍